epi/src/lib.rs

```diff
@@ -144,7 +144,14 @@ pub trait App {
     /// where `APPNAME` is what is returned by [`Self::name()`].
     fn save(&mut self, _storage: &mut dyn Storage) {}
 
-    /// Called once on shutdown (before or after [`Self::save`])
+    /// Called before a shutdown that _can_ be aborted. This means that if you are on the web or
+    /// the window is forcefully closed, this method will not be called.
+    fn on_pre_exit(&mut self) -> bool {
+        true
+    }
```

Comment:
Need to document what return value means and maybe provide an example (from an issue?)

Reply:
I have added an example and hopefully the documentation is now more clear.